x/gamm/keeper/keeper.go

```diff
@@ -50,7 +50,7 @@ func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey, paramSpace paramtyp
 		paramSpace = paramSpace.WithKeyTable(types.ParamKeyTable())
 	}
 	return Keeper{
-		storeKey:   storeKey,
+		StoreKey:   storeKey,
```

Comment:
Note: not sure how secure it is to make `StoreKey` public as we need it outside of this package. Would really appreciate any guidance or feedback here

Reply:
We definitely can't do this

Reply:
I see, is there an alternative to this then because i need StoreKey in `query_proto_wrap.go` ([LINK](https://github.com/osmosis-labs/osmosis/pull/2586/files#diff-94cc9713a7cdae08c57c24a5bf3ae01c62696eb0c99d4831854e973330f9eab4R65))

Reply:
Oh the pagination requires store key and the store. 
Proposed option here is to create a query method that returns the pagination as well